sql/item_mods.sql

```diff
@@ -58092,7 +58211,7 @@ INSERT INTO `item_mods` VALUES (26108,165,5); -- CRITHITRATE: 5
 -- Snotra Earring
 INSERT INTO `item_mods` VALUES (26109,13,8);  -- MND: 8
 INSERT INTO `item_mods` VALUES (26109,30,10); -- MACC: 10
--- TODO Enfeeble Magic Duration: +10%
+-- TODO: Enfeeble Magic Duration: +10%
```

Comment:
I thought we had a modifier for this already. However, easy to implement aswell.

This is a note for "future me", btw, feel free to ignore this particular comment.

Reply:
Unfortunately, at this point, these are the only enfeeble mods I see:
```
	Line  165:     ENFEEBLE  = 114, // Enfeebling Magic Skill
	Line  301:     // Resistances to enfeebles - Job Traits/Job Abilities/Atmas/Items/Gear
	Line  320:     // MEVA bonus to enfeebles (Bar-Effect, for example. And modifiers in mobs)
```